src/braket/pennylane_plugin/translation.py

```diff
@@ -445,10 +445,10 @@ def _(op: ParametrizedEvolution, _parameters, device=None):
     # The driven wires aren't the same as `op.wires` as `op.wires` contains
     # all device wires due to interaction term.
     pulse_wires = qml.wires.Wires.all_wires([pulse.wires for pulse in pulses])
-    mapped_wires = pulse_wires.map(device.wire_map)
+    frames = {w: device._device.frames[f"q{w}_drive"] for w in pulse_wires}
```

Comment:
With this change, we don't need the `BraketDevice` instance for the translation, just the `AwsDevice` instance, so we could change that as well.